chain-impl-mockchain/src/leadership/genesis/vrfeval.rs

```diff
@@ -65,6 +65,12 @@ impl TryFrom<Milli> for ActiveSlotsCoeff {
     }
 }
 
+impl Into<Milli> for ActiveSlotsCoeff {
```

Comment:
I always thought the consensus was to implement `From<ActiveSlotCoeff> for Milli` rather than implementing `Into<Milli> for ActiveSlotCoeff`. Since one implies the other.

Reply:
Yes, prever implementing `From` whenever you can. It can always be done also in the crate defining the parameter type.